lineage/states/StateDistributionGaussian.py

```diff
@@ -85,7 +86,6 @@ def censor_lineage(self, censor_condition, full_list_of_gens, full_lineage, **kw
 
         output_lineage = []
         for gen_minus_1, level in enumerate(full_list_of_gens[1:]):
-            true_gen = gen_minus_1 + 1  # generations are 1-indexed
             for cell in level:
                 basic_censor(cell)
                 if cell.observed:
```

Comment:
As a quick fix, I wrote this bad censoring function for the Gaussian state distribution file, but ideally it should have no censor function (or a censor function that doesn't censor). We can deal with this on a new branch.